navigation.json

```
@@ -39,7 +39,7 @@
       "text": "components.containers.footer.links.codeOfConduct"
     },
     {
-      "link": "https://github.com/nodejs/node/blob/HEAD/SECURITY.md#security",
+      "link": "https://github.com/nodejs/node/blob/main/SECURITY.md",
```

this should link to the durable GitHub link
```suggestion
      "link": "https://github.com/nodejs/node/security",
```

see https://github.com/nodejs/nodejs.org/pull/5995 for when we did that elsewhere. I agree the fragment is a bit odd

---

Should this go to `https://github.com/nodejs/node/security/policy` then, rather than `https://github.com/nodejs/node/security`, to match the others (albeit they still use the anchor too, which seems unnecessary)

---

the content looks the same so i guess i dont understand too much of the difference. 

`https://github.com/nodejs/node/security` is shorter and on the top-level nav - whereas policy is the more-squirrelled away one.

🤷‍♂️

---

I'll use the shorter one and update the others for consistency 👍

---

Hm, maybe I should just back this out and create a follow-up issue. There are a _lot_ of usages of the longer `https://github.com/nodejs/node/security/policy` with and without the anchor, including a lot of security release blog posts. Thoughts?

---

@ljharb you helped author the original changes, do you have more insight here that could help us?

---

Do we have a consensu yet? @bmuenzenmeyer did you reach out Jordan on Slack?

---

@ovflowd I did not, no. Perhaps the security WG would be a better resource

---

@ljharb says "the shorter link seems newer (i believe it didn’t show the policy a few months ago), but also lacks the left-hand tab navigation, so the longer one still seems a better choice to me. Either way, I’d suggest using the longer one now, and change them all in a follow-up if that’s desired."

---

👍 Will roll with the longer one for consistency